.chloggen/add_authentication_user_subnamespace.yaml

```diff
@@ -0,0 +1,23 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
```

Comment:
should this be "breaking" for users who replaced `user.id` for the deprecated `enduser.id`?

Reply:
I think deprecated attributes should also be updated to use `user.authentication.id` instead of `user.id`
Also schema-next should reflect the same.

Reply:
I have updated it. please check out general.yaml.